docker-app/qfieldcloud/core/paginators.py

```diff
@@ -35,3 +36,7 @@ def count(self):
             else:
                 return estimate
         return len(self.object_list)
+
+
+class LimitOffsetPagination(pagination.LimitOffsetPagination):
+    pass
```

Comment:
I think the class should live in another file, called `pagination.py`, because they are a concept of DRF, and the current file is implementing concepts from django-admin.